gtsfm/utils/io.py

```diff
@@ -36,16 +39,18 @@ def load_image(img_path: str) -> Image:
     """
     original_image = PILImage.open(img_path)
 
-    exif_data = original_image.getexif()
+    exif_data = original_image._getexif()
```

Comment:
is there an alternate for this?

Reply:
There is. This was more a hotfix. The question is whether we want to work with "flattened" or hierarchical EXIF: https://github.com/python-pillow/Pillow/pull/4947/files

May require more discussion as to what would be cleanest. The new Pillow EXIF API is not well documented, IMHO.